### PR TITLE
Background mode

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Local notifications when test end and app is in background.
Test reminder and auto run tests in background.

### CODE REVIEW

File: app/src/main/java/org/openobservatory/ooniprobe/ooniprobeApp.java [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
 import android.app.Application;
+
+import org.openobservatory.ooniprobe.activity.MainActivity;
+
 import uk.co.chrisjenx.calligraphy.CalligraphyConfig;
 
 public class ooniprobeApp extends Application {
-
+    //for further reference http://stackoverflow.com/questions/11411395/how-to-get-current-foreground-activity-context-in-android
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/ooniprobeApp.java]: Perhaps this comment should be closer to the code it actually refers to?

Comment (lorenzoPrimi) [inline app/src/main/java/org/openobservatory/ooniprobe/ooniprobeApp.java]: It needs to be there. I will need it for the new UI

**Comment by bassosimone** [comment]
@lorenzoPrimi The app does not compile. I get this error:

```
$HOME/src/github.com/measurement-kit/ooniprobe-android/app/src/main/java/org/openobservatory/ooniprobe/utils/AlarmReceiver.java:19: error: cannot find symbol
        MainActivity currentActivity = ((ooniprobeApp)context.getApplicationContext()).getCurrentActivity();
                                                                                      ^
  symbol:   method getCurrentActivity()
  location: class ooniprobeApp

```

**Comment by bassosimone** [comment]
@lorenzoPrimi I have further tested the pull request. Unless there is some urgency, I'd like to do little more testing tomorrow morning and merge, otherwise feel free to merge yourself. 👍

**Pull request merged**